src/status_im/ui2/screens/chat/components/message_home_item/view.cljs

```diff
@@ -74,31 +74,31 @@
    (if (:ens-verified contact)
      [rn/view {:style {:margin-left 5 :margin-top 4}}
       [icons/icon :main-icons2/verified {:no-color true
-                                         :size 12
-                                         :color (colors/theme-colors colors/success-50 colors/success-60)}]]
+                                         :size     12
+                                         :color    (colors/theme-colors colors/success-50 colors/success-60)}]]
      (when (:added? contact)
        [rn/view {:style {:margin-left 5 :margin-top 4}}
         [icons/icon :main-icons2/contact {:no-color true
-                                          :size 12
-                                          :color (colors/theme-colors colors/primary-50 colors/primary-60)}]]))
+                                          :size     12
+                                          :color    (colors/theme-colors colors/primary-50 colors/primary-60)}]]))
    [text/text {:style (style/timestamp)}
     (time/to-short-str timestamp)]])
 
 (defn messages-home-item [item]
   (let [{:keys [chat-id color group-chat last-message timestamp name unviewed-mentions-count unviewed-messages-count]} item
-        display-name (if-not group-chat (first (<sub [:contacts/contact-two-names-by-identity chat-id])) name)
-        contact      (when-not group-chat (<sub [:contacts/contact-by-address chat-id]))
-        photo-path   (when-not (empty? (:images contact)) (<sub [:chats/photo-path chat-id]))]
+        display-name (if-not group-chat (first (rf/sub [:contacts/contact-two-names-by-identity chat-id])) name)
```

Comment:
I like to think of it as one-to-one chats being the default case, so I start with `not group-chat`. But I can change it tho.

Reply:
I think for this particular example its better to keep it `if-not` to be consistent with the other declarations form. Below it we have `when-not group-chat` which would otherwise have to be `contact (if group-chat nil (<sub [:contacts/contact-by-address chat-id]))`

Reply:
I haven't looked it that way, but as I commented, it's a nitpick, it's not really important/blocker. By all means, keep the code as is if you prefer.